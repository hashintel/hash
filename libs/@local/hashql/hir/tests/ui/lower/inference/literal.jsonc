//@ run: pass
//@ description: Checks type inference for an integer literal.
{ "#literal": 123 }
