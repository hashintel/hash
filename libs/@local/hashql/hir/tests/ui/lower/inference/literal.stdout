123

--------------------------------------

123

--------------------------------------

Node:
123


Type:
Integer